draft-ietf-mops-streaming-opcons.md

```diff
@@ -765,6 +753,16 @@ For most of the history of the Internet, we have trusted UDP-based applications
 
 In recent times, the usage of UDP-based applications that were not simple query-response protocols has grown substantially, and since UDP does not provide any feedback mechanism to senders to help limit impacts on other users, application-level protocols such as RTP {{RFC3550}} have been responsible for the decisions that TCP-based applications have delegated to TCP - what to send, how much to send, and when to send it. So, the way some UDP-based applications interact with other users has changed.
 
+In contrast to adaptive segmented delivery as described in {{adapt-deliver}}, several applications use unreliable UDP or SCTP with its "partial reliability" extension {{RFC3758}} to deliver Media encapsulated in RTP {{RFC3550}} or raw MPEG Transport Stream ("MPEG-TS")-formatted video {{MPEG-TS}}, when the media is being delivered in situations such as broadcast and live streaming, that better tolerate occasional packet loss without retransmission.
```

Comment:
I agree it's reasonable to move this content to this section, but I think it might need a bit more refactoring to read well.

I also noticed SRT also has a partial reliability feature, I believe?  Maybe it would make sense to combine with the 2nd paragraph above, and maybe to restructure slightly, here's a suggestion:

---

In contrast to adaptive segmented delivery as described in {{adapt-deliver}}, some applications use an unreliable transport for streaming media.  There are a variety of approaches, including:

 - raw MPEG Transport Stream ("MPEG-TS")-formatted video {{MPEG-TS}} over UDP, which makes no attempt to account for reordering or loss in the transport
 - RTP {{RFC3550}}, which can notice loss and repair some limited reordering
 - SCTP {{RFC4960}}, which can use partial reliability {{RFC3758}} to recover from some loss, but can abandon recovery to limit head-of-line blocking
 - SRT {{SRT}}, which can use forward error correction and time-bound retransmission to recover from loss within certain limits, but can abandon recovery to limit head-of-line blocking

Under congestion and loss, approaches like the above generally experiences transient video artifacts more often and delay of playback effects less often, as compared with reliable segment transport. Often one of the key goals of using a UDP-based transport that allows some unreliability is to reduce latency to better support applications like videoconferencing, or for other live-action video with interactive components, such as some sporting events.

Congestion avoidance strategies for deployments using unreliable transport protocols vary widely in practice, ranging from being entirely unresponsive to congestion, to using feedback signaling to change encoder settings (as in {{RFC5762}}), to using fewer enhancement layers (as in {{RFC6190}}), to using proprietary methods to detect "quality of experience" issues and turn off video in order to allow less bandwidth-intensive media such as audio to be delivered.

More details about congestion avoidance strategies used with unreliable transport protocols are included in {{udp-behavior}}.

---

It also might be appropriate to remove the preceding paragraph (the "In recent times, UDP started getting used even though it doesn't have congestion control" one--I think at least its use of RTP collides with the later bits).